src/Components/Common.js

```diff
@@ -52,4 +53,15 @@ const RBACPermissions = {
     write: ['malware-detection:*:*']
 };
 
-export { link, gqlProps, isBeta, totalMatchesTitle, expandMatchMetadata, capitalize, strong, RBACPermissions };
+// eslint-disable-next-line max-len
+const documentationURL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_reporting_malware_signatures_on_rhel_systems/index';
+const documentationLink = linkText => (
```

Comment:
Generally, if something returns a component it should be a component. Meaning that this should look more like this:

```js
const DocumentationLink = ({linkText}) => (
```

And it should be called like a component:

```js
<DocumentationLink linkText={intl.formatMessage(messages.learnMore))} />
```

What we could do in this case as well is make use of children and not need a `linkText` prop.


```js
const DocumentationLink = ({children}) => (
...
<a href={documentationURL} target="__blank" rel="noopener noreferrer">
         {children} <ExternalLinkAltIcon />
     </a>
...
```


```js
<DocumentationLink>
 {intl.formatMessage(messages.learnMore))}
</DocumentationLink>
```

Reply:
Ok, thanks for that @bastilian.  I'll make those changes